pom.xml

```diff
@@ -385,7 +441,7 @@
       <plugin>
         <groupId>com.github.spotbugs</groupId>
         <artifactId>spotbugs-maven-plugin</artifactId>
-        <version>${spotbugs.version}</version>
+        <version>4.4.2.2</version>
```

Comment:
Does this intentionally update only the maven plugin and not `spotbugs` and `spotbugs-annotations`?

Reply:
Yes, for now. Spotbugs 4.5 fails with messages about exposed arrays. I didn't want to address that in this PR as it is big enough by itself. The maven plugin update is necessary for Java 17.